crates/bevy_sprite/src/sprite.rs

```diff
@@ -38,6 +44,32 @@ impl Sprite {
             ..Default::default()
         }
     }
+
+    /// Create a sprite from an image
+    pub fn from_image(image: Handle<Image>) -> Self {
+        Self {
+            image,
+            ..Default::default()
+        }
+    }
+
+    /// Create a sprite from an image, with an associated texture atlas
+    pub fn from_atlas_image(image: Handle<Image>, atlas: TextureAtlas) -> Self {
```

Comment:
Is it confusing that it's called `from_atlas_image` and the argument order is image then atlas?

Reply:
I was thinking "atlas" as an adjective but I'm not attached to the name

Reply:
It's probably fine, it just made me pause for a sec